<launch>
    <arg name="matcher" default="sift" />

    <node pkg="jsk_2014_picking_challenge" type="fake_camera_info_publisher.py" name="fake_camera_info_publisher" output="screen"></node>
    <node pkg="jsk_2014_picking_challenge" type="image_publish_server.py" name="image_publish_server" output="screen"></node>

    <include file="$(find jsk_2014_picking_challenge)/launch/$(arg matcher)_matcher.launch">
        <arg name="input_image" value="/image_publish_server/output" />
        <arg name="input_info" value="/fake_camera_info_publisher/output" />
        <arg name="respawn" value="true" />
    </include>

    <node pkg="jsk_2014_picking_challenge" type="test_object_matching.py" name="test_object_matching" output="screen" respawn="true">
        <param name="matcher" type="string" value="$(arg matcher)" />
    </node>
</launch>
